.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: UI Tests
+
+on:
+    workflow_dispatch:
+    deployment_status:
+
+env:
+  GIT_PAT: ${{ secrets.GIT_PAT }}
+  MNEMONIC: ${{ secrets.MNEMONIC }}
+  MNEMONIC_MAINNET: ${{ secrets.MNEMONIC_MAINNET }}
```

Comment:
We don't use this anymore

Reply:
Why? Just David updated the `MNEMONIC` _mnemonic_ in the secrets and the tests worked with the correct wallet.
 also need `GIT_PAT` so that the tests can run after _deploy_preview_

Reply:
From all these envs we need only:
- `GIT_PAT`
- `MNEMONIC`

And for bridge project we also need:
- `MNEMONIC_METAMASK`
- `PRIVATE_KEY_METAMASK`

For other envs, it should be safe to remove them.

Reply:
> Why? Just David updated the MNEMONIC mnemonic in the secrets and the tests worked with the correct wallet.
also need GIT_PAT so that the tests can run after deploy_preview

I think I chose wrong lines in github, sorry for the inconvenience

Reply:
Got it about `git_pat`